webservice/constants.py

```diff
@@ -1,4 +1,4 @@
-GSE_DATA_LENGTH = 79  # 4(packet_time) + 15(bool) + 4 * 14(floats) + 4(crc)
+GSE_DATA_LENGTH = 83  # 4(packet_time) + 15(bool) + 4 * 14(floats) + 4(crc)
```

Comment:
```suggestion
GSE_DATA_LENGTH = 83  # 4(packet_time) + 15(bool) + 4 * 15(floats) + 4(crc)
```
chore: update comment